Core/Core/Planner/CalendarToDo/ViewModel/EditCalendarToDoViewModel.swift

```diff
@@ -117,10 +179,37 @@ final class EditCalendarToDoViewModel: ObservableObject {
                 }
             }
             .store(in: &subscriptions)
+    }
 
-        calendarListProviderInteractor
-            .load(ignoreCache: false)
-            .sink()
+    private func subscribeIsTouched<T: Equatable>(to publisher: Published<T>.Publisher) {
+        publisher
+            .removeDuplicates()
+            .dropFirst()
+            .sink { [weak self] _ in self?.isTouched = true }
             .store(in: &subscriptions)
     }
+
+    private func saveAction() -> AnyPublisher<Result<Void, Error>, Never> {
```

Comment:
This return type seems a bit more complex than it should be. What about `AnyPulisher<Void, Error>` ?

Reply:
It's reworked without `mapToResult()`